Directory.Build.props

```diff
@@ -21,15 +21,16 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
+    <LangVersion>12.0</LangVersion>
   </PropertyGroup>
 
   <!-- Shared Package Versions -->
   <PropertyGroup>
-    <OrleansVersion>7.2.3</OrleansVersion>
-    <StreamUtilsVersion>13.1.3</StreamUtilsVersion>
+    <OrleansVersion>8.0.0</OrleansVersion>
+    <StreamUtilsVersion>!!!!USE-DEV!!!!</StreamUtilsVersion>
```

Comment:
consume dev utils package after this has been released
https://github.com/jonathansant/Orleans.Streams.Utils/pull/22/files